olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -151,6 +151,22 @@ spec:
        * with the ability to select a specific subset of events from a particular
          producer.
 
+
+    ## Knative Kafka
+
+    Knative Kafka is a **[Technology Preview feature](https://access.redhat.com/support/offerings/techpreview)!**
+
+    Apache Kafka is a distributed streaming platform and Knative Kafka adds
+    the following components:
+
+    1. A Knative Eventing Source to consume messages from an Apache Kafka topic
+    1. A Knative Eventing Channel that is backed by Apache Kafka topics
+
+    ### Knative Kafka and Knative Eventing
+    Knative Eventing and Apache Kafka is required since this operator will attempt to
```

Comment:
Maybe `Apache Kafka` could be more specific to RedHat's offering? Strimzi?

Reply:
yeah, not sure what's the right wording - I will ask Simon for input

Reply:
How about something like 

```
An Apache Kafka such as [AMQ Streams](https://access.redhat.com/documentation/en-us/red_hat_amq/7.7/html/amq_streams_on_openshift_overview/index) is required to be accessible from the cluster. This  operator will create Knative components which work with the Apache Kafka cluster.
```?